cli/kaos_cli/services/tests/test_state_service.py

```diff
@@ -0,0 +1,228 @@
+import os
+import time
+from parameterized import parameterized
+from configobj import ConfigObj
+
+from unittest import TestCase
+
+from kaos_cli.services.state_service import StateService
+from kaos_cli.constants import CONFIG_PATH, KAOS_STATE_DIR
+
+
+def state_service_test_inputs():
+    return [
+        (
+            'context1', 'param1'
+        ),
+        (
+            'context2', 'param2'
+        ),
+        (
+            '', 'param3'
+        ),
+        (
+            'context4', ''
+        ),
+        (
+            '', ''
+        ),
+        (
+            {}, 'param6'
+        ),
+        (
+            'context7', {}
+        ),
+        (
+            {}, {}
+        ),
+        (
+            '', {}
+        ),
+        (
+            {}, ''
+        ),
+        (
+            None, 'param11'
+        ),
+        (
+            'context11', None
+        ),
+        (
+            None, None
+        ),
+        (
+            {}, None
+        ),
+        (
+            None, {}
+        ),
+        (
+            '', None
+        ),
+        (
+            None, ''
+        ),
+        (
+            1, 'param18'
+        ),
+        (
+            'context', 2
+        ),
+        (
+            3, {}
+        ),
+        (
+            {}, 4
+        ),
+        (
+            5, ''
+        ),
+        (
+            '', 6
+        ),
+        (
+            7, None
+        ),
+        (
+            None, 8
+        ),
+
+    ]
+
+
+class TestStateService(TestCase):
+
+    # Arrange
+    TestCase.state_service = StateService()
+    TestCase.switch = False
+
+    def test_state_service_create(self):
+
+        # Act
+        TestCase.state_service.create()
```

Comment:
Maybe `self.state_service.create()`?

Reply:
This has been changed, as the services are now instantiated for each test